README.md

```diff
@@ -89,7 +89,19 @@ Run tests
 ```bash
 python -m unittest
 ```
-
+## Calculating Code Coverage
+Install Coverage.py
+```bash
+pip install coverage
+```
+Calculate code coverage
+```bash
+coverage run -m
```

Comment:
Please give some examples. What does the parameter `-m` refer to? Which modules/scripts did you test?